### DISCUSSION TITLE
MC-707 Wishlist for Roland in 2024

### DISCUSSION THREAD

**Discussion by ricardofeynman** [ricardofeynman/Awesome-MC-707#1] [Ideas]
r/, Discord channels and other sites have plenty of feature request conversations spread out over many threads. Hopefully this can be a place for such discussions to be easily accessible for the community and for Roland to keep track of.

**Comment by ricardofeynman** [comment]
OK I'll go first. My own feature requests for the MC-707:

**1. Abilty to use external MIDI controller to rapidly switch EXT/PC/TRK1-8/MIXOUT for Scatter FX.** 
Ideally, we'd be able to map pads/buttons on an external controller to allow scatters on individual tracks/EXT/PC/MIX at the same time. e.g. Scatter A on pad 1 effects track 1, Scatter B on pad 2 effects MIXOUT, Scatter C on pad 3 effects track 1, and so on. 

**2. Ability to use external MIDI controller to rapidly switch between MULTI FX.** (Elucidate!)

**3. Tempo automation.**  (Elucidate!)

**4. Allow us to do awesome things like this again** https://www.benis.it/cm/articoli/Fantom_Personalmodel_EN.htm 
That was a painful firmware upgrade and potentially put a lot of new customers off as it signalled Roland was anti-community value added DIY. Why? Not everyone's going to dive for those after sale packs (especially since they've shot up in price so tremendously over time). Those who want (and can afford) a Fantom grade product are a different subset of Roland's customer base.
 
As I'm hoping that at some stage someone from Roland will have eyes on this thread, I'm going to expand on my reasoning.
 
While I appreciate that from the Corpo top brass perspective, this DIY approach may be seen as coming into conflict with the Roland Cloud/Zenology Pro business model, I don't see why DIY should be forcefully excluded. Giving MC owners, who may not be ready to part with cash for all the bells and whistles, the benefit of letting them sample some of the possibilities available by way of creative community endeavours like this could drive future sales.
 
I think it would entice more owners/potential new customers if they can experience the personal models on the MC-707, follow the articles that advanced users such as benis67 has provided, and learn to create something of value to share back. Whether or not this new(ish)ly imposed limitation exists, I don't think it will be the deciding factor that 'encourages' MC users to purchase a Fantom - they're entirely different beasts with their own sets of pros/cons and price points - but it's a good promotion of the expanded capabilities of other devices.

**5. SysEx.** 
I mean come on Roland. You want to shift more units? A rewrite for SysEx in v2.0. Open this beast up for the nerds. Everyone will benefit. Would it really be so hard? It seems to me these devices were a bit of a gamble by Roland. I'd love to see a flourishing community for the MC's, but I think you'll likely need a fresh crop of third party devs doing additional marketing for these devices' (hopefully soon-to-be) expanded capabilities to get the increased buzz they deserve. 

With Akai now dropping well executed AI features like MPC Stems, I think there's a strong arguement to be made to the Corpos that there are renewed benefits to opening the MC's up for the community, and that it's a really good moment to take some increased measures which will maximize the value their existing hardware has to offer.

**Addendum:**

I think I'll just keep editing this post as I go. Depending on how many other people contribute their own wishlists here, I'll start sifting through my bookmarked threads and copy over feature requests that I'm also keen on seeing as a priority for future upgrades.

Please add your 2cents here if you want to increase the likeliness that Roland won't skip another year of firmware updates!

**Reply by Locriana** [reply]
Hi,
About 5: I figured out MC-707 Sys-Ex recently - at least in the scope of general communication and the zen-core:
https://github.com/Locriana/mcpoker
The MV-1 is similar, but it can communicate request-response over USB MIDI only.
I included some more information in the description to my YT video and in its comment section:
https://youtu.be/x_kHTL99cvk
